src/board_controller/openbci/ganglion.cpp

```diff
@@ -372,36 +434,62 @@ int Ganglion::config_board (char *config)
         {
             return res;
         }
-        // call itself with disabled streaming
-        res = config_board (config);
+        /* hack, in ganglion commands to enable/disable impedance check control data streaming
+        to handle it and keep it consistent with other devices we swap chars for command_start and
+        command_stop
+        */
+        if ((strlen (config)) && (config[0] == 'z'))
+        {
+            start_command = "z";
+            stop_command = "Z";
+        }
+        else
+        {
+            if ((strlen (config)) && (config[0] == 'Z'))
+            {
+                start_command = "b";
+                stop_command = "s";
```

Comment:
I suggest declaring "b" "s" "z" and "Z" as named constants somewhere. And make a couple functions to switch the start/stop commands from standart to impedance (because i see it done in two places)